libraries/core-react/stories/components/Search.stories.tsx

```diff
@@ -70,105 +70,103 @@ export const Examples: Story<SearchProps> = () => {
     handleOnBlur()
   }
 
+  return (
+    <div
+      style={{
+        background: isFocused ? 'cyan' : 'transparent',
+      }}
+    >
+      <Typography variant="h4" as="h2">
+        I am connected to the search input
+      </Typography>
+      <Search
+        aria-label="Focused and not focused"
+        id="search-focusedAndNot"
+        placeholder="Search"
+        onChange={handleOnChange}
+        onFocus={handleFocus}
+        onBlur={handleBlur}
+      />
+    </div>
+  )
+}
+WithOnFocusAndBlur.storyName = 'With on focus and blur'
+
+export const WithPredefinedValue: Story<SearchProps> = () => (
+  <Search
+    aria-label="predefined"
+    id="search-predefined"
+    placeholder="Search"
+    onChange={handleOnChange}
+    defaultValue="Predefined value"
+  />
+)
+WithPredefinedValue.storyName = 'With predefined value'
+
+export const CenteredAndStyled: Story<SearchProps> = () => (
+  <Wrapper>
+    50% width
+    <StyledSearch
+      aria-label="contained"
+      id="search-contained"
+      placeholder="Search"
+      onChange={handleOnChange}
+    />
+  </Wrapper>
+)
+
+CenteredAndStyled.storyName = 'Centered and styled'
+
+export const InsideAForm: Story<SearchProps> = () => (
+  <form action="/">
+    <Search placeholder="Search" onChange={handleOnChange} />
+  </form>
+)
+
+export const Disabled: Story<SearchProps> = () => (
+  <Search
+    aria-label="disabled"
+    id="search-disabled"
+    placeholder="Search"
+    onChange={handleOnChange}
+    onFocus={handleOnFocus}
+    onBlur={handleOnBlur}
+    disabled
+  />
+)
+InsideAForm.storyName = 'Inside a form element'
```

Comment:
Suggest moving up to next to variable declaration

Reply:
I like it when it's close to the actual story. You can't move it up anyways, because you can't use it before declaration. 

If you want them together, it has to be at the very bottom of the file. I will accept PR's if it is of the utmost importance to you. 🥇

Reply:
Was thinking moving it up to line `125`, which is the actual story? Now its under disabled...?